pkg/workceptor/kubernetes.go

```diff
@@ -467,14 +467,7 @@ mainLoop:
 				case containerState.Terminated != nil:
 					// We got EOF and the pod terminated in a bad state, we will return the error and mark the job as failed.
 					if containerState.Terminated.ExitCode != 0 {
-						kw.GetWorkceptor().nc.GetLogger().Info("Container in %s pod has terminated, with nonzero exit code: %v", WorkerContainerName, containerState.Terminated.ExitCode)
-						*stdoutErr = fmt.Errorf("detected Error: %s for pod %s/%s. Pod has terminated, with terminated exit code: %v", err,
-							podNamespace,
-							podName,
-							containerState.Terminated.ExitCode,
-						)
-
-						return
+						kw.GetWorkceptor().nc.GetLogger().Info("Container in %s pod has terminated, with nonzero exit code: %v, treminated reason: %v and terminated message: %v", WorkerContainerName, containerState.Terminated.ExitCode, containerState.Terminated.Reason, containerState.Terminated.Message)
 					}
 
 					// EOF and exit code is 0, this is a good state, we need to check if last line has data
```

Comment:
I think this comment is out of date - we have EOF and terminated, don't care what the exit code was now.